sdk/guestconfiguration/Microsoft.Azure.Management.GuestConfiguration/src/Generated/Models/GuestConfigurationNavigation.cs

```diff
@@ -44,18 +44,30 @@ public GuestConfigurationNavigation()
         /// configuration package is uploaded.</param>
         /// <param name="contentHash">Combined hash of the guest configuration
         /// package and configuration parameters.</param>
+        /// <param name="assignmentType">Specifies the assignment type and
+        /// execution of the configuration. Possible values are Audit,
+        /// DeployAndAutoCorrect, ApplyAndAutoCorrect and ApplyAndMonitor.
+        /// Possible values include: 'Audit', 'DeployAndAutoCorrect',
+        /// 'ApplyAndAutoCorrect', 'ApplyAndMonitor'</param>
+        /// <param name="contentType">Specifies the content type of the
+        /// configuration. Possible values could be Builtin or Custom.</param>
         /// <param name="configurationParameter">The configuration parameters
         /// for the guest configuration.</param>
+        /// <param name="configurationProtectedParameter">The protected
+        /// configuration parameters for the guest configuration.</param>
         /// <param name="configurationSetting">The configuration setting for
         /// the guest configuration.</param>
-        public GuestConfigurationNavigation(string kind = default(string), string name = default(string), string version = default(string), string contentUri = default(string), string contentHash = default(string), IList<ConfigurationParameter> configurationParameter = default(IList<ConfigurationParameter>), ConfigurationSetting configurationSetting = default(ConfigurationSetting))
+        public GuestConfigurationNavigation(string kind = default(string), string name = default(string), string version = default(string), string contentUri = default(string), string contentHash = default(string), string assignmentType = default(string), string contentType = default(string), IList<ConfigurationParameter> configurationParameter = default(IList<ConfigurationParameter>), IList<ConfigurationParameter> configurationProtectedParameter = default(IList<ConfigurationParameter>), ConfigurationSetting configurationSetting = default(ConfigurationSetting))
```

Comment:
As interface changes caused not backward compatible, please increate the major version of the package, not minor version

Reply:
sure. done